src/api/utils/initial_data.py

```diff
@@ -47,48 +49,106 @@
                                  description="Mentoring students outside of"
                                  " Andela e.g. via SheLovesCode",
                                  value=250)
+activity_types = [
+    interview, open_saturdays, tech_event, open_source, hackathon, blog,
+    app, mentor, marketing, press, outside_mentoring
+]
 
 # societies
 phoenix = Society(name="Phoenix")
 istelle = Society(name="iStelle")
 sparks = Society(name="Sparks")
 invictus = Society(name="Invictus")
+societies = [phoenix, istelle, sparks, invictus]
 
 # test countries
 kenya = Country(name='Kenya')
 
-# test user
-user = User(
-    uuid="-KdQsMt2U0ixIy_-yJEH",
+# cohorts
+cohort_14_ke = Cohort(name='Cohort 14', country=kenya)
+
+## users
+# member
+member = User(
+    uuid="-KdQsMtixI2U0y_-yJEH",
     name="Test User",
     photo="https://lh6.googleusercontent.com/-1DhBLOJentg/AAAAAAAAA"
-          "AI/AAAAAAAAABc/ImM13eP_cAI/photo.jpg?sz=50",
-    email="lawrence.wachira@andela.com",
+          "AI/AAAAAAAAABc/ImeP_cAI/photo.jpg?sz=50",
+    email="test.user.societies@andela.com",
+    country=kenya,
+    cohort=cohort_14_ke,
+    society=phoenix
+)
+
+# president
+president = User(
+    uuid="-KdQsMtixG4U0y_-yJEH",
+    name="Test President",
+    photo="https://lh6.googleusercontent.com/-1DhBLOJentg/AAAAAAAAA"
+          "AI/AAAAAAnAABc/ImeP_cAI/photo.jpg?sz=50",
+    email="test.president.societies@andela.com",
+    role="president",
     country=kenya,
+    cohort=cohort_14_ke,
     society=phoenix
 )
 
+# success ops
+success_ops = User(
+    uuid="-KdQsMtixG4U0y_-yJEF",
+    name="Test Success Ops",
+    photo="https://lh6.googleusercontent.com/-1DhBLOJentg/AAAAAAAAA"
+          "AI/AAAAAAnAABc/ImeP_cAI/photo.jpg?sz=50",
+    email="test.successops.societies@andela.com",
+    role="success ops",
```

Comment:
Same here.
The role column isn't needed since we get the role from the payload.

Reply:
Same as above. The models on develop don't have a separate roles table.

Reply:
Psst. One thing you could also do is merge this PR to develop. 😆
That way, you can just rebase and update accordingly.